llvm/test/Transforms/LoopVectorize/as_cast.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: opt -passes=loop-vectorize %s
+; ModuleID = '<bc file>'
+target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-unknown-linux-gnu"
+
+define void @foo(ptr addrspace(1) %in) {
+entry:
+  br label %for.body.i.epil
+
+for.body.i.epil:                                  ; preds = %for.body.i.epil, %entry
```

Comment:
Please clean up the IR names here and elsewhere to make the test easier to read, e.g %loop here, drop various epil. prefixes etc